README.md

```diff
@@ -52,7 +53,58 @@ sudo apt update && sudo apt upgrade
 
 After that  just install the same packages as above.
 
-### Build
+------
+
+## Build
+
+There are two ways to compile eitehr with CMake or with meson.
+By default meson is faster, but if your project uses CMake, those instructions might be better for integration.
+
+### Building with meson
```

Comment:
Could you please move Meson section after CMake section? CMake support is prioritized. At least for now due to better IDE integration than Meson projects.

Reply:
Fixed in 6b1a7eb